src/index.html

```diff
@@ -12,11 +12,111 @@
     />
     <title>Moyo header</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
       rel="stylesheet"
-      href="./style.css"
+    />
+    <link
+      rel="stylesheet"
+      href="style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
```

Comment:
The `href="#"` attribute is used as a placeholder. If these links are intended to navigate to specific sections or pages, consider replacing `#` with the actual URLs or section IDs.